drivers/watchdog/Kconfig

```diff
@@ -11,8 +11,12 @@ menuconfig WATCHDOG
 
 if WATCHDOG
 
+config HAS_WDT_DISABLE_AT_BOOT
+	bool
+
```

Comment:
any help section to explain the purpose of this CONFIG ?

Reply:
I think its purpose is pretty obvious, just like in case of `HAS_WDT_MULTISTAGE` where we also have no help entry.